ToDoExercise/Content/app.js

```diff
@@ -0,0 +1,86 @@
+﻿
+var app = new Vue({
+
+    el: "#app",
+    components: {
+        litask,
+        modalcomponent,
+        draggable: window['vuedraggable']      //esto se hace ya que vuedraggable lo utilizamos a través de cdn, con lo que no había otra forma de utilizarlo...
+    },
+    data() {
+        return {
+            modalInput: {},
+            forms: [],
+            counter: 0
+        }
+    },
+    methods: {
+
+        createCard() {
+            this.incrementCounter()
+            this.forms.push(this.createEmptyCardObject())
+            this.saveLocal();
+        },
+        deleteTaskFromId(id) {
+            this.forms.splice(this.getIndexFromId(id), 1)
```

Comment:
esto es raro, debería quedar algo así:
this.tasks = this.tasks.filter((element) => element.id !== taskId)